_includes/navbar.html

```diff
@@ -2,7 +2,9 @@
     <div class="container">
         <a href="#" class="nav-opener"><span>menu</span></a>
         <div class="logo">
-            <a href="/"><img src="/images/logo.png" height="29" width="122" alt="Evenly"></a>
+            <a href="/">
+                <img src="/images/logo.svg" onerror="this.onerror=null; this.src='/images/logo.png'" height="29" width="122" alt="Evenly">
```

Comment:
Does a browser prefer srcset to src? If so would it not be best to do `<img src="/images/logo.png" srcset="<img src="/images/logo.svg"... `